pkg/client/config/config.go

```diff
@@ -40,7 +40,7 @@ func init() {
 		"Paths to a kubeconfig. Only required if out-of-cluster.")
 
 	// This flag is deprecated, it'll be removed in a future iteration, please switch to --kubeconfig.
-	flag.StringVar(&apiServerURL, "master", "",
+	flag.StringVar(&apiServerURL, "apiServerURL", "",
```

Comment:
This requires a ⚠️ and it's breaking-release. We should just removed the flag

Reply:
Let's do separate these in two PRs, one for this change that removes the flag entirely, and one for the comments

Reply:
ok thanks

Reply:
done